packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -93,6 +93,9 @@ const Primary = styled.div`
     display: flex;
     justify-content: flex-start;
     align-items: center;
+    @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+        width: calc(100% - 48px);
+    }
```

Comment:
what does this do? I see no difference on small screen.

Reply:
I tried to reduce wrapper width for some space for close button, but it seems like text overflows anyway, so removing in 6c919e8